README.md

```diff
@@ -1,32 +1,33 @@
-# Puck
-
-The visual editor for React.
-
-<p align="left">
-  <a aria-label="NPM version" href="https://www.npmjs.com/package/@measured/puck">
-    <img alt="" src="https://img.shields.io/npm/v/@measured/puck.svg?style=for-the-badge&labelColor=000000">
-  </a>
-  <a aria-label="License" href="https://github.com/measuredco/puck/blob/main/LICENSE">
-    <img alt="" src="https://img.shields.io/npm/l/@measured/puck.svg?style=for-the-badge&labelColor=000000">
-  </a>
-  <a aria-label="Join the community on Discord" href="https://discord.gg/D9e4E3MQVZ">
-    <img alt="" src="https://img.shields.io/badge/Join%20the%20Discord-blueviolet.svg?style=for-the-badge&logo=Discord&labelColor=000000&logoWidth=20">
-  </a>
-  <a aria-label="Browse the awesome-puck community repo" href="https://github.com/measuredco/awesome-puck">
-    <img alt="" src="https://img.shields.io/badge/repo-awesome--puck-fc60a8.svg?style=for-the-badge&labelColor=000000&logoWidth=20">
-  </a>
-  <a aria-label="Contributing guide" href="https://github.com/puckeditor/puck/blob/main/CONTRIBUTING.md">
-    <img alt="" src="https://img.shields.io/badge/contributions-welcome-blue?style=for-the-badge&labelColor=000000">
-  </a>
-</p>
-
-## Demo
-
-Visit https://demo.puckeditor.com/edit to try the demo.
-
-## Documentation
-
-Visit https://puckeditor.com to view the full documentation.
+<br /><br /><br />
+
+<div align="center">
+
+<a href="https://puckeditor.com">
+  <picture>
```

Comment:
Final comment - can you make this max 768px and centre align? Less overwhelming and most projects do this

Preview -

<img width="1133" height="794" alt="Image" src="https://github.com/user-attachments/assets/d4929688-2dc4-4fa8-8016-8e85c6b4b402" />